### PR TITLE
port and short options

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

**Comment by kzidane** [comment]
https://github.com/cs50/ide50/issues/198

File: files/home/ubuntu/.cs50/bin/flask [hunk 1]
```diff
@@ -24,8 +24,14 @@ if [[ "$1" == "run" ]]; then
     do
         if [[ "$1" =~ ^--host= ]]; then
             host="$1"
+        elif [[ "$1" == "-h" ]]; then
+            host="$1 $2"
+            shift 2
```
Comment (dmalan) [inline files/home/ubuntu/.cs50/bin/flask]: Should this def be `shift 2` given that we still have a second shift on [L42](https://github.com/cs50/ide50/pull/230/files#diff-a616842b4f2f23c9ca59e11c59296990R42)?

File: files/home/ubuntu/.cs50/bin/flask [hunk 2]
```diff
@@ -24,8 +24,14 @@ if [[ "$1" == "run" ]]; then
...
         elif [[ "$1" =~ ^--port= ]]; then
             port="$1"
+        elif [[ "$1" == "-p" ]]; then
+            port="$1 $2"
+            shift 2
```
Comment (dmalan) [inline files/home/ubuntu/.cs50/bin/flask]: Here too?

File: files/home/ubuntu/.cs50/bin/flask [hunk 3]
```diff
@@ -36,6 +42,9 @@ if [[ "$1" == "run" ]]; then
         shift
     done
 
+    # kill any process listing on the specified port
+    fuser --kill "${port//[^0-9]}/tcp" &> /dev/null
```
Comment (dmalan) [inline files/home/ubuntu/.cs50/bin/flask]: Does this trim everything that's not a number, thereby leaving just the port #?

File: files/home/ubuntu/.cs50/bin/flask (hunk 1 above)
Comment (kzidane) [reply files/home/ubuntu/.cs50/bin/flask]: Oops, you're right. Fixed!

Comment (dmalan) [reply files/home/ubuntu/.cs50/bin/flask]: code reviews ftw!

File: files/home/ubuntu/.cs50/bin/flask (hunk 3 above)
Comment (kzidane) [reply files/home/ubuntu/.cs50/bin/flask]: Yes, basically searches for anything that matches `[^0-9]` and removes it. So should remove anything that's not a digit.

**Pull request merged**